lib/rules/no-invalid-double-slash-comments/index.mjs

```diff
@@ -33,9 +33,9 @@ const rule = (primary) => {
 		});
 
 		root.walkRules((ruleNode) => {
-			// ruleNode.selectors elements are trimmed
-			const selectors = ruleNode.selector.split(',');
-			const ruleStringified = ruleNode.toString();
+			const selectors = (ruleNode.raws?.selector?.raw ?? ruleNode.selector).split(',');
```

Comment:
getRuleSelector util seems available instead.

Reply:
Thank you @ybiquitous 🙇

Maybe interesting to have some custom eslint rules to check such stuff :)